titration/utils/devices/syringe_pump_mock.py

```diff
@@ -117,4 +115,4 @@ def drive_step_stick(self, cycles, direction):
             else:
                 return int(temp)
         else:
-            interfaces.lcd.print("Arduino Unavailable", 4, constants.LCD_CENT_JUST)
+            interfaces.lcd.print("Arduino Unavailable", 4, 2)
```

Comment:
Should this be "center" instead of 2?

Reply:
Yes, I just missed that changed. Should be updated now.